src/api/python/speechd_config/config.py

```diff
@@ -602,7 +602,15 @@ def create_user_configuration(self):
                     return
         # Copy the original intact configuration files
         # creating a conf/ subdirectory
-        shutil.copytree(buildconfig.SPD_CONF_ORIG_PATH, self.test.user_conf_dir())
+        config_root = self.test.user_conf_dir()
+        shutil.copytree(buildconfig.SPD_CONF_ORIG_PATH, config_root)
+        # Ensure the files are writeable when copying from immutable directory.
+        os.chmod(self.test.user_conf_dir() , 0o755)
+        for root, dirs, files in os.walk(self.test.user_conf_dir()):
+            for d in dirs:
+                os.chmod(os.path.join(root, d), 0o755)
+            for f in files:
+                os.chmod(os.path.join(root, f), 0o644)
```

Comment:
I'd say make sure to `and` with the current umask? Otherwise that could set broader permissions than what the user configured her environment to set by default.